stress/shell/src/main/java/alluxio/stress/cli/client/StressClientIOBench.java

```diff
@@ -164,9 +184,121 @@ public ClientIOTaskResult runLocal() throws Exception {
     ClientIOTaskResult.ThreadCountResult result = context.getResult();
     LOG.info(String.format("thread count: %d, errors: %d, IO throughput (MB/s): %f", numThreads,
         result.getErrors().size(), result.getIOMBps()));
+
     return result;
   }
 
+  /**
+   * Read the log file from java agent log file.
+   *
+   * @param clientIOTaskResult client io task result
+   * @return ClientIOTaskResult with java agent info
+   * @throws IOException
+   */
+  @SuppressFBWarnings(value = "DMI_HARDCODED_ABSOLUTE_FILENAME")
+  public synchronized ClientIOTaskResult addAdditionalResult(ClientIOTaskResult clientIOTaskResult)
+      throws IOException {
+    Map<String, PartialResultStatistic> methodNameToHistogram = new HashMap<>();
+
+    try (final BufferedReader reader = new BufferedReader(new FileReader(AGENT_OUTPUT_PATH))) {
+      String line;
+
+      final ObjectMapper objectMapper = new ObjectMapper();
+      while ((line = reader.readLine()) != null) {
+        final Map<String, Object> lineMap;
+        try {
+          lineMap = objectMapper.readValue(line, Map.class);
+        } catch (JsonParseException e) {
```

Comment:
this may need to catch additional ones like: https://github.com/Alluxio/alluxio/blob/master/stress/shell/src/main/java/alluxio/stress/cli/StressMasterBench.java#L243